docs/source/development/linting.md

```diff
@@ -40,20 +41,28 @@ pip install black
 pip install flake8
 pip install isort
 ```
+#### Configure `flake8`
+
+Store your `flake8` configuration in a file named `setup.cfg` within your project root. The Kedro starters use the [following configuration](https://github.com/kedro-org/kedro-starters/blob/main/pandas-iris/%7B%7B%20cookiecutter.repo_name%20%7D%7D/setup.cfg):
+
+```text
+[flake8]
+max-line-length=88
+extend-ignore=E203
+```
 
-### Run linting tools
+### Run the tools
 Use the following commands to run lint checks:
 ```bash
 black --check <project_root>
-flake8 <project_root>
```

Comment:
Is there a reason you don't want to keep this here?

Reply:
Good spot. I meant to put it separately.